src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,45 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack {
+    static Scanner input = new Scanner(System.in);
+
+/*    protected void addlack(Lack lack,User user) {
+        if (checkIfForwarder(user)) {
```

Comment:
Tutaj trzeba ten item stworzyć, tak naprawdę, bo go jeszcze nie ma więc musiałoby być
 System.out.println("Give item name: ");
lack.item.itemName = input.next();
i później to samo z itemID